src/schedule/zephyr_ll.c

```diff
@@ -252,6 +254,17 @@ static void zephyr_ll_run(void *data)
 		       NOTIFIER_TARGET_CORE_LOCAL, NULL, 0);
 }
 
+static void schedule_ll_callback(void *data)
```

Comment:
there already have a similar ccount calculation, not sure you can directly use it:
https://github.com/thesofproject/sof/blob/main/src/schedule/zephyr_ll.c#L137C1-L137C1

line 137 - line 146, it also use ccount for zephyr, same data, you may need modify this part if you want to use it.

Reply:
Yes, it's probably also based on telemetry from reference firmware. The difference is, the old one does not have all the necessary metrics and it outputs to trace. Mine puts output to memory window and its compatible with reference firmware.

As for ccount for zephyr, seems like i need to actually use the zephyr one so I'll take that from the old one.